Godeps/Godeps.json

```diff
@@ -10,6 +10,9 @@
 			"ImportPath": "github.com/antchfx/xpath",
 			"Rev": "1f3266e7730781347c9decd229c1fe1c18be760c"
 		},
+			"ImportPath": "github.com/dutchcoders/go-clamd",
```

Comment:
This project doesn't currently support clamd's FILDES method which passes a file descriptor across the socket using the `SCM_RIGHTS` message type. Because of this and because our clamd will not be able to scan by paths (since it's in a different fs namespace), it's going to have to stream the data across the socket (necessitating another copy of the data). This is something we were hoping to avoid. I think we should either:

1. Send a PR to the project that adds FILDES support
or
2. Use https://github.com/Miciah/clam-scanner instead. It was purpose built by us to solve this problem (since when I looked for a golang clamd front-end, I failed to find anything). You can see the FILDES code in action here:

https://github.com/Miciah/clam-scanner/blob/f34ad6de7f7afff39452e0e11f8549939e067b06/pkg/clamav/session.go#L317-L326

His `ScanPath()` function uses filepath.Walk() to scan an entire sub-tree.